run-suite-task/ghost-inspector/services/index.ts

```diff
@@ -46,11 +46,11 @@ export class Suite {
 
   async execute (): Promise<boolean> {
     // TODO: test network failure
-    console.log('POSTing execute request with body', this.suiteExecuteUrl, this.body)
+    const safeUrl = `${this.suiteResultsBaseUrl}/${suiteResultId}/?apiKey=***`
```

Comment:
This doesn't seem like the right URL... I think you copy/pasted from the `poll()` function, but the endpoints are different. Shouldn't it be `${this.suiteExecuteBaseUrl}/${this.request.suiteId}/execute/?apiKey=`?

Also might be cleaner to reuse the `suiteExecuteUrl` function and just add a `obfuscate` boolean parameter that hides the API key when true.

Reply:
Oh good call, I was being lazy there.

Reply:
This is fixed, [tests added](https://github.com/ghost-inspector/ghost-inspector-vsts-extension/pull/9/files#diff-82b3f263f3caec26e54bcd2f1a8f66d3R26)